islandora_solr_config/includes/admin.inc

```diff
@@ -110,6 +111,12 @@ function islandora_solr_config_admin_rss_settings($form, &$form_state) {
     '#description' => t('Email address for person responsible for technical issues relating to channel.'),
     '#default_value' => $rss_channel['webMaster'] ? $rss_channel['webMaster'] : '',
   );
+  $form['rss_limit'] = array(
+    '#type' => 'textfield',
+    '#title' => 'RSS results limit',
+    '#description' => 'Maximum number of results via RSS.',
+    '#default_value' => $rss_limit,
+  );
```

Comment:
Is this a paged value? Or the total max? Just wondering

Reply:
Can you actually page RSS results? As far as I know it's a maximum; same as the original method (which is paged for the regular search results view but not RSS).

Reply:
Yes you can. Out of scope here? Was not even an option before right? https://tools.ietf.org/html/rfc5005

Reply:
OK, that's something I definitely want, but it's another ticket for sure.